resources/campaigns/attributes/cmp_is_draft.yml

```diff
@@ -0,0 +1,2 @@
+description: Whether or not the campaign is still a draft.
```

Comment:
See my comment in the campaign model: you don't need to extrapolate these attributes into separate files. You can leave them inline.